src/main/java/com/GG/T9AgeCombat/predicates/DetermineModificationPredicate.java

```diff
@@ -12,7 +12,7 @@ private DetermineModificationPredicate() {
 
     public static void applyBonus(Unit unit, String modification, int value) {
         if (modification.equals(STRENGTH.toString())) {
-            unit.updateStrength(value);
+            unit.updateStrengthModifier(value);
```

Comment:
so I am not understanding why @nonfinal was given to all the stats when this is the pattern?  I didn't see anywhere that the base strength of a unit was being modifieid